Library/Homebrew/test/utils/git_spec.rb

```diff
@@ -0,0 +1,175 @@
+require "utils/git"
+
+describe Git do
+  before(:all) do
+    git = HOMEBREW_SHIMS_PATH/"scm/git"
+    file = "lib/blah.rb"
+    repo = Pathname.new("repo")
+    FileUtils.mkpath("repo/lib")
+    `#{git} init`
```

Comment:
What's failing about them? It'd be good to fix that, rather than ignore the failure using backticks.